setup.py

```diff
@@ -24,9 +30,9 @@
     author_email="ahearin@anl.gov",
     description="Tools for generating synthetic cosmological data",
     long_description="Tools for generating synthetic cosmological data",
-    install_requires=["numpy", "cython"],
+    install_requires=DEPENDENCIES,
```

Comment:
You want to separately use `setup_requires` vs `install_requires` to distinguish requirements at build vs runtime.